pkg/dialer/factory.go

```diff
@@ -152,24 +152,25 @@ func (f *Factory) clusterDialer(clusterName, address string) (dialer.Dialer, err
 			return cd(network, address)
 		}, nil
 	}
-	logrus.Debugf("dialerFactory: no tunnel session found for cluster [%s]", cluster.Name)
+	logrus.Debugf("dialerFactory: no tunnel session found for cluster [%s], falling back to nodeDialer", cluster.Name)
 
-	if cluster.Status.Driver != v3.ClusterDriverRKE {
```

Comment:
It's a legacy restriction from the times where we didn't start node agents in imported clusters